README.md

```diff
@@ -184,6 +184,17 @@ aws cloudformation deploy \
   --parameter-overrides PipelineStackName=mellon-image-service-pipeline Receivers=me@myhost.com
 ```
 
+How to build the Manifest Pipeline Pipleine
+```console
+aws cloudformation deploy \
+  --capabilities CAPABILITY_NAMED_IAM \
+  --stack-name mellon-manifest-pipeline \
+  --template-file deploy/cloudformation/manifest-pipeline-pipeline.yml \
+  --tags Name='mellon-manifest-pipeline' Contact='lib-esu-group@nd.edu' Owner='jhartzle' Description='CF for AppStream Lambda Function and Api Gateway.' \
```

Comment:
We've been generalizing the READMEs - with things like `Owner='myid'` and `Contact=me@myhost.com` for sharing purposes. It's probably worth the change esp. for Mellon for the sake of consistency.